app/views/browse/dewey.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Browse by Dewey Call Number</h1>
```

Comment:
You'd rename this partial to `_dewey` if we put classification as a parameter and not in the named path.

Reply:
done